### PR TITLE
DROOLS-2107: Run Scenario against rules that use inheritance

### PR DESCRIPTION
**Pull request by jomarko**

### CODE REVIEW

File: drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/resources/org/drools/workbench/screens/testscenario/backend/server/HelloEveryOne.gdst [hunk 1]
```diff
@@ -83,7 +83,11 @@
     <entries/>
   </auditLog>
   <imports>
-    <imports/>
+    <imports>
+      <org.kie.soup.project.datamodel.imports.Import>
```
Comment (manstis) [inline drools-wb-screens/drools-wb-test-scenario-editor/drools-wb-test-scenario-editor-backend/src/test/resources/org/drools/workbench/screens/testscenario/backend/server/HelloEveryOne.gdst]: I know it's unrelated to this PR; but do you have an older (pre- ```kie-soup```) GDST with an import?

I thought we "aliased" the FQCN to the simple-name however if we do not; and older GDST contain the old FQCN we may have a problem loading such files in newer versions of the workbench. I had a look here but can only find "old" GDST without any imports.

**Comment by jomarko** [comment]
@manstis I am going to add similar tests with different characteristics, see https://issues.jboss.org/browse/BXMSQE-258. Feel free to create task/s there for me.

**Comment by manstis** [comment]
@jomarko I assume this can be merged now?

**Comment by jomarko** [comment]
@manstis yes, I think so.

**Pull request merged**